File: ServiceSandbox/app/src/main/java/njit/mt/servicesandbox/MainActivity.java
package njit.mt.servicesandbox;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }
    public void startService(View view){
        Intent intent = new Intent(this, HelloThereService.class);
        startService(intent);
    }
    public void stopService(View view){
        Intent intent = new Intent(this, HelloThereService.class);
        stopService(intent);
    }
}

File: Unit5/TapItScores/app/src/main/java/com/ethereallab/tapitscores/MainActivity.kt
package com.ethereallab.tapitscores

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.ethereallab.tapitscores.databinding.ActivityMainBinding


class MainActivity : AppCompatActivity() {

    private lateinit var binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)

        binding.btnStart.setOnClickListener {
            startActivity(Intent(this, GameActivity::class.java))
        }

        binding.btnScores.setOnClickListener {
            startActivity(Intent(this, ScoresActivity::class.java))
        }
    }
}